tmva/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
+find_package(LAPACK)
```

Comment:
Why do you need this? 
 We have already 
`find_package(BLAS)` in `cmake/modules/SearchInstalledSoftware.cmake` and it should be enough

Reply:
Ahh yes, removed it now. Thanks!